### PR TITLE
Feature/add example with go router

### PR DESCRIPTION
**Pull request by junddao**
## Status

Ready for review

## Breaking Changes

NO

## Description

Added go_router example code using bloc.

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: examples/flutter_todos/pubspec.lock [hunk 1]
```diff
@@ -5,26 +5,26 @@ packages:
...
+    version: "5.6.0"
   args:
     dependency: transitive
     description:
       name: args
-      sha256: "139d809800a412ebb26a3892da228b2d0ba36f0ef5d9a82166e5e52ec8d61611"
+      sha256: "4cab82a83ffef80b262ddedf47a0a8e56ee6fbf7fe21e6e768b02792034dd440"
       url: "https://pub.dev"
     source: hosted
-    version: "2.3.2"
```
Comment (tenhobi) [inline examples/flutter_todos/pubspec.lock:27]: Any reason to modify another's example pubspec lock? :)

Comment (junddao) [reply examples/flutter_todos/pubspec.lock]: Sorry. It's my mistake. I update my PR except pubspec.lock file in todo project. @tenhobi

**Comment by felangel** [comment]
Hi @junddao 👋 
Thanks for the PR but at this time I'd prefer not to add more examples since they are quite time-consuming to maintain and we'd likely also want to write up a proper tutorial in the documentation. In this case, I'd highly recommend publishing your example as a personal repo under your GitHub account and I'm more than happy to point people to it, thanks!

**Comment by junddao** [comment]
> Hi @junddao 👋 Thanks for the PR but at this time I'd prefer not to add more examples since they are quite time-consuming to maintain and we'd likely also want to write up a proper tutorial in the documentation. In this case, I'd highly recommend publishing your example as a personal repo under your GitHub account and I'm more than happy to point people to it, thanks!

@felangel  🙂
Thank you for your reply. 
I will publish on my repository this example.
Next time, have a chance to help you, I will do.

have a good day.

**Pull request closed without merging**